docs/features/readonly-variables-and-parameters.md

```diff
@@ -0,0 +1,61 @@
+### High Level Language Proposal:  
+Allow 'readonly' to be used on local variables and on parameters. 
+
+'readonly' on local variables and 'parameters' will work similarly as it does for 'fields' (with important differences listed in the next paragraph).  Namely, 'readonly' local variables and parameters cannot be written to in any way (i.e. through assignment, use of ```++/+=```, passing as a 'ref/out', etc.). 
+
+Important differences:
+unlike 'readonly' fields, 'readonly' locals must have an initializer.
+unlike 'readonly' fields (which can be written to multiple times in constructors), 'readonly' locals can only have the initializer and can never be written to again.
```

Comment:
that's a good case.  Will def have to discuss at the LDM.  I can see why that would be problematic.